test/unit/specs/util/options_spec.js

```diff
@@ -282,7 +282,8 @@ describe('Util - Option merging', function () {
     var f1 = function () {}
     var f2 = function () {}
     var f3 = function () {}
-    var componentA = { template: 'foo', methods: { f1: f1, f2: function () {} } }
```

Comment:
It throws an error:
```js
TypeError: Object prototype may only be an Object or null: undefined
```
I'm working on that, but I cannot face the problem. Any idea? :)